subsys/bluetooth/mesh/lightness_srv.c

```diff
@@ -972,6 +966,7 @@ const struct bt_mesh_model_cb _bt_mesh_lightness_srv_cb = {
 #ifdef CONFIG_BT_SETTINGS
 	.settings_set = bt_mesh_lightness_srv_settings_set,
 	.start = bt_mesh_lightness_srv_start,
+	.pending_store = lightness_srv_pending_store,
```

Comment:
Should be aligned with respect to `bt_mesh` prefix or not in these functions. I see the convention across files is a bit anarchy at the moment, but in a single file they should be aligned at least.